Assignment-1-July-18/index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>JSON Identifier</title>
+    <h1>Welcome to JSON Identifier</h1>
+    <link rel="stylesheet" href="style.css" />
+    <script>
+      function jsonChecker(text) {
```

Comment:
**Recommendations**
use anonymous functions instead of named function, use named function where its absolute necessary.